internal/bgp/bgpfrr/config.go

```diff
@@ -0,0 +1,121 @@
+package bgpfrr
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"text/template"
+)
+
+// TODO configFileName will need to be set to the filename of the FRR configuration
+// file in the container.
+var configFileName = "frr.conf"
+
+const configTemplate = `
+log stdout {{.Loglevel}}
+hostname {{.Hostname}}
+
+{{range .Routers -}}
+router bgp {{.MyASN}}
+  no bgp default ipv4-unicast
+  bgp router-id {{.RouterId}}
+{{range .Neighbors }}
+  neighbor {{.Addr}} remote-as {{.ASN}}
```

Comment:
This renders a line like this:

```
neighbor 10.89.0.5:179 remote-as 64512
```

which is malformed resulting in an invalid configuration:

```
2021-09-27T20:12:31.573021851Z line 13: Failure to communicate[13] to bgpd, line:  neighbor 10.89.0.5:179 remote-as 64512
```

The port number needs either to be stripped out (defaults to port 179) or a new line to override default port needs to be set. For example:

```
neighbor 10.89.0.5 remote-as 64512
neighbor 10.89.0.5 port 9179
neighbor 10.89.0.6 remote-as 64512
```

Reply:
Thanks